clients/typescript/Makefile

```
@@ -1,3 +1,6 @@
+build_generator:
+	(cd ../../generator && make deps && make build)
```

This can be rewritten using the following more concise style
```suggestion
	$(MAKE) -C build
```
Note that `build` already depends on `deps` in generator's Makefile.